welcome/about-kueue.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="about-kueue"]
+= About {product-title}
+:context: about-kueue
+
+toc::[]
+
+{product-title} is a Kubernetes-native system that manages access to resources for jobs.
+{product-title} can determine when a job waits, is admitted to start by creating pods, or should be _preempted_, meaning that active pods for that job are deleted.
+
+[NOTE]
+====
+In the context of {product-title}, a job can be defined as a one-time or on-demand task that runs to completion.
+====
+
+{product-title} is based on the link:https://kueue.sigs.k8s.io/docs/[Kueue] open source project.
+
+{product-title} is compatible with environments that use heterogeneous, elastic resources. This means that the environment has many different resource types, and those resources are capable of dynamic scaling.
+
+{product-title} does not replace any existing components in a Kubernetes cluster, but instead integrates with the existing Kubernetes API server, scheduler, and cluster autoscaler components.
+
+{product-title} supports all-or-nothing semantics. This means that either an entire change is applied successfully to pods, or the entire change is rolled back, preventing partial updates and issues that could be caused by them.
```

Comment:
```suggestion
{product-title} supports all-or-nothing semantics. This means that a workload can be admitted with all of its components or it would be rejected if the workload can not fit on the cluster.
```

Something like that.